bin/yb-ctl

```diff
@@ -497,6 +507,11 @@ class ClusterOptions:
                                    "separated by commas.")
             self.placement_info.append(t_item)
 
+        num_drives = int(getattr(args, "num_drives", 1))
+        if num_drives <= 0:
+            raise ExitWithError("Invalid number of drives: {}".format(num_drives))
+        self.drives = ["disk-{}".format(i) for i in xrange(1, num_drives + 1)]
```

Comment:
It would be good to move away from xrange to be one step closer to Python 3 compatibility.